MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -159,14 +166,17 @@ void LayeredVolume::removeCongruentElements(std::size_t nLayers, std::size_t nEl
 			{
 				delete _elements[upper_offset+j];
 				_elements[upper_offset+j] = nullptr;
+				_materials[upper_offset+j] = -1;
```

Comment:
The special value `-1` (and also the above inserted `0` entries) are not documented.
There is also some `noDataReplacementValue`, which might be appropriate instead of the `-1`, but I'm not sure here.